src/onboarding/create-utils.js

```diff
@@ -39,3 +40,27 @@ export function prepareTransactionCreatorFromAbi(abi, toAddress) {
     }
   }
 }
+
+export async function getGasPrice() {
+  const web3 = getWeb3(web3Providers.default)
+  try {
+    const response = await fetch(
+      'https://ethgasstation.info/json/ethgasAPI.json',
+      {
+        method: 'GET',
+        mode: 'cors',
+        cache: 'no-cache',
+      }
+    )
+    const jsonResponse = await response.json()
+    const priceInWei = web3.utils.toWei(
+      jsonResponse.gasPriceRange[40].toString(), // using '40' as a safe price, this should be set via user input
+      'gwei'
+    )
+    return priceInWei
```

Comment:
We may want to add a bit more logic here.

I'd say if the price is currently lower than 3 gwei, we should just set it to 3 to avoid "nasty" surprises if the market jumps up. The user should always have the choice in the signing provider to adjust this if he thinks it's too expensive (not always true, but I'd blame the provider if not).